tests/test_cli_download_files.py

```diff
@@ -57,27 +63,40 @@ def test_download_files_filter_regexp():
     if os.path.isfile(test_file):
         os.remove(test_file)
     test_download_files_filter = CliRunner()
-    test_result_regexp = test_download_files_filter.invoke(
+    test_result_regexp_one = test_download_files_filter.invoke(
         download_files, ["--recid", 3005, "--filter-regexp", "py$"]
     )
-    assert test_result_regexp.exit_code == 0
+    assert test_result_regexp_one.exit_code == 0
     assert os.path.isfile(test_file) is True
-    assert test_result_regexp.output.endswith("\nDownload completed!\n")
+    assert test_result_regexp_one.output.endswith("\nDownload completed!\n")
     if os.path.isfile(test_file):
         os.remove(test_file)
+    test_result_regexp_two = test_download_files_filter.invoke(
+        download_files, ["--recid", 5500, "--filter-regexp", "0"]
```

Comment:
To test non-existent regexp for files, please use longer string such as "notexisting" or "wontmatchanything" or something... Just in case there could be an update to record 5500 that would bring a file containing 0 inside.